www/components/AgendaIcon.js

```diff
@@ -0,0 +1,17 @@
+import CalendarTodayRoundedIcon from '@material-ui/icons/CalendarTodayRounded';
+import AccessAlarmRoundedIcon from '@material-ui/icons/AccessAlarmRounded';
+import styles from "../styles/AgendaIcon.module.css"
+function AgendaIcon() {
+    return (
+        <div className={styles.contain}>
+            <CalendarTodayRoundedIcon style={{fontSize: 35,
+            color: "var(--agendaSecondary)"}}/>
```

Comment:
```suggestion
              color: "var(--agendaSecondary)"}}/>
```
When its a property of a component its good to indent so that is is immediately apparent you aren't looking at an element.

Reply:
prettier would have done that.. this is because of eslint

Reply:
i might have disabled eslint for this one file because when i logged back on it fixed this and other problems